### PR TITLE
Use GitHub OAuth role from GitHub

### PR DESCRIPTION
**Pull request by jsf9k**
Remove it from this repository.

If the other devs like this approach then we can extract a lot of other roles that are likely even more reusable than `github_oauth`.

### CODE REVIEW

**Review by felddy — APPROVED** [review]
I like these changes, and would be happy to see us adopt it across all of our projects. So dry!

**Comment by jsf9k** [comment]
I'd still like @dav3r and @KyleEvers to weigh in before merging.  I too think that in most cases separate and reusable Ansible roles are the way forward, but I want others to voice concerns if they have them.

**Comment by KyleEvers** [comment]
I also like these changes. Looks good.

File: README.md [hunk 1]
```diff
@@ -1,32 +1,46 @@
...
+because the AMIs produced by those lines are large and need extra time
+to be copied, as discussed
 [here](https://github.com/hashicorp/packer/issues/6536#issuecomment-407925535).
 
-Build Terraform-based infrastructure with:
+Also note that
+```
+ansible-galaxy install --force -r packer/ansible/requirements.yml
+```
+will update the roles that are being pull from external sources.  This
```
Comment (dav3r) [inline README.md]:
```suggestion
will update the roles that are being pulled from external sources.  This
```

**Review by dav3r — CHANGES_REQUESTED** [review]
I'm a big fan of this approach- let's keep it going. 👍

**Review by dav3r — APPROVED** [review]

**Pull request merged**